build.sh

```diff
@@ -157,8 +157,13 @@ else
     cmake ..
 fi
 
-# Build the sources with the number of available cores
-cmake --build . -- -j "$(nproc)"
+# If MAKEFLAGS is unset, we set it to "-j$(nproc)"
+if [[ -z "${MAKEFLAGS}" ]]; then
+    export MAKEFLAGS="-j$(nproc)"
```

Comment:
Why do we need to export them?

Reply:
To not duplicate the `cmake` command in the `if` statement